src/index.js

```diff
@@ -96,24 +98,133 @@ export async function getArgs () {
       console.log(result)
     })
 
+  program
+    .command('installAppBundle <InstalledAppId> <AgentHash> <AppBundleSource>')
+    .alias('b')
+    .option('-e --membraneProof <membraneProof>', 'provide membrane proof for app bundle install and runtime validation - should be paired with --cellNick')
+    .option('-n --cellNick <cellNick>', 'provide cell nick - should be paired with --membraneProof')
+    .option('-u --uid <uid>', 'provide uid to happ')
+    .description('install provided happ bundle with given id and details: calls InstallAppBundle(installed_app_id, agent_key, source, membrane_proofs?, uid?) -> { installed_app_id, cell_data: [ { cell_id, cell_nick } ], status: { inactive: { reason: [Object] } } }')
+    .action(async (InstalledAppId, AgentHash, AppBundleSource, { membraneProof, cellNick, uid }) => {
+      let membraneProofs
+      // throw error if one, but now both, of the membraneProof and CellNick are present - both are needed to form the membrane_proofs object
+      if (!membraneProof ^ !cellNick) {
```

Comment:
```suggestion
      if (!(membraneProof || cellNick)) {
```

Feel free to ignore this suggestion, but I find it easier to read written this way. It reads as an english sentence "if not membraneProof or cellNick" whereas using the exclusive or version, there isn't a simple translation into english so I have to do more thinking to read it.